taxcalc/functions.py

```diff
@@ -1604,60 +1583,36 @@ def C1040(e07400, e07180, e07200, c07220, c07230, e07250,
 
 
 @iterate_jit(nopython=True)
-def DEITC(c08795, c59660, c09200, c07100, c08800, c05800, _othertax):
+def DEITC(c08795, c59660, c09200, c07100, c08800, c05800, _avail, _othertax):
 
     # Decomposition of EITC
-
-    if c08795 > 0 and c59660 > 0 and c08795 <= c59660:
-        c59680 = c08795
-
-        _comb = c59660 - c59680
-
-    else:
-
-        c59680 = 0.
-        _comb = 0.
-
-    if c08795 > 0 and c59660 > 0 and c08795 > c59660:
-        c59680 = c59660
+    _comb = 0
+    c10950 = 0
+    c59680 = min(c59660, max(0, _avail))
```

Comment:
It seems that `max(0, _avail)` is used to evaluate `c59680` instead of `c08795`.